### PR TITLE
Update subjects-guide.md

### LINKED ISSUE #15: опис для ЕБЦЗ

### LINKED ISSUE #16: опис для ОЗСЖ

### LINKED ISSUE #18: опис для англ мови

### LINKED ISSUE #19: опис для матану

### PR DESCRIPTION
**Pull request by kulchitskyi**
Added description for MATAN, ecologichna bezpeka, english, OZSG
close #19 
close #18 
close #16 
close #15

### CODE REVIEW

File: src/guides/subjects-guide.md [hunk 1]
```diff
@@ -258,3 +275,13 @@ P.S. Перелік викладачів може змінюватись, але
...
+ Всього у вас буде 6 тем, по 3 теми на семетр. Для того щоб закрити тему вам необхідно здати розрахункову роботу, це завдання які вам необхідно виконати впродовж місяця, та модульну контрольну роботу - ті ж самі завдання що і на розрахунковій, але за 1,5 години. Ці види робіт оцінюються в 6-7 балів РР і 10 балів МКР, також у кінці семестру є екзамен який оцінюється у 50 балів та на якому будуть усі теми.
+На першому курсі у вас буде два викладача:
+
+Лектор - Легеза Віктор Петрович
+Викладач чудово знає свій предмет, при необхідності пояснить, що стало не зрозуміло, але зрозуміліше від цього не стане. Ще готуйтеся що, якщо ваше питання здаватиметься Віктору Петровичу надто легким, можете потрапити на шквал гніву.  
+
+Практик - Нещадим Олександр Михайлович
```
Comment (JackVorobey23) [inline src/guides/subjects-guide.md:286]: add tag ### for lectors/practitioners

File: src/guides/subjects-guide.md [hunk 2]
```diff
@@ -246,7 +256,14 @@ P.S. Перелік викладачів може змінюватись, але
...
 **Навички, які Вам потрібні:** вміти прочитати текст англійською, мати базу з мови
 
-Топ, найкращий предмет з усіх, рекомендую
+Топ, найкращий предмет з усіх, рекомендую. 
+Будете робити завдання з класруму на 1.5 бала (це основа ваших балів з предмету)
+За кожен семестр треба зробити 2 презентації на вільну тему англійською, вони теж оцінюються в 5 балів. Також за будь-яку роботу на парі (діалог, підготовлена розповідь, розповідь про дз) отримуєте максимум по 2.5 бали.
+Можете також прийняти участь в Студентській конференції (за бали), це  найулюбленіша тема викладачки, може говорити про неї пів пари (на кожній парі протягом семестру)
+В кінці семестру буде Файнал тест на 10 балів. 
+
+Викладачка - Шепелєва Олена Володимирівна
```
Comment (JackVorobey23) [inline src/guides/subjects-guide.md:265]: add ###

File: src/guides/subjects-guide.md [hunk 3]
```diff
@@ -226,6 +228,14 @@ P.S. Перелік викладачів може змінюватись, але
 
 **Навички, які Вам потрібні:** ніякі (все пояснюється в матеріалах від викладача).
 
+Досить цікавий предмет для загального розвитку, але іноді матеріал викладання є досить сумнівним, тому краще сприймати його як загальну інформацію. 
+Будуть лекції, де буде викладатись матеріал, та практики, де викладач буде опитувати Вас за темою лекції(за кожну практику можна отримати максимум 2 бали). Питання до практик Ви перед парою зможете знайти на платформі Moodle, до якої Вас доєднають на початку року. Там також інколи будуть даватись додаткові тести(зазвичай за темами практик і оцінюватимуться у 2 бали).
+Наприкінці деяких тем будуть МКР(це тести на 10-20 питань, які зазвичай беруться з практик) і оцінюватимуться у 20-25 балів. 
+
+Лектор та практик  – Сога Сергій Михайлович
```
Comment (JackVorobey23) [inline src/guides/subjects-guide.md:235]: add ###

**Review by JackVorobey23 — CHANGES_REQUESTED** [review]

File: src/guides/subjects-guide.md [hunk 4]
```diff
@@ -258,3 +275,13 @@ P.S. Перелік викладачів може змінюватись, але
...
+ Всього у вас буде 6 тем, по 3 теми на семетр. Для того щоб закрити тему вам необхідно здати розрахункову роботу, це завдання які вам необхідно виконати впродовж місяця, та модульну контрольну роботу - ті ж самі завдання що і на розрахунковій, але за 1,5 години. Ці види робіт оцінюються в 6-7 балів РР і 10 балів МКР, також у кінці семестру є екзамен який оцінюється у 50 балів та на якому будуть усі теми.
+На першому курсі у вас буде два викладача:
+
+Лектор - Легеза Віктор Петрович
+Викладач чудово знає свій предмет, при необхідності пояснить, що стало не зрозуміло, але зрозуміліше від цього не стане. Ще готуйтеся що, якщо ваше питання здаватиметься Віктору Петровичу надто легким, можете потрапити на шквал гніву.
```
Comment (JackVorobey23) [inline src/guides/subjects-guide.md:284]: image

**Review by JackVorobey23 — CHANGES_REQUESTED** [review]

File: src/guides/subjects-guide.md (hunk 4 above)
Comment (traewe) [reply src/guides/subjects-guide.md:284]: image

**Review by anzhhm — APPROVED** [review]

**Review by JackVorobey23 — APPROVED** [review]

**Review by JackVorobey23 — APPROVED** [review]

**Review by JackVorobey23 — APPROVED** [review]

**Pull request merged**